src/components/Gallery/PhotoCarousel.jsx

```diff
@@ -33,6 +33,16 @@ import hothX_image3 from '../../images/gallery-carousel/hoth-X/image3.jpg';
 import hothX_image4 from '../../images/gallery-carousel/hoth-X/image4.jpg';
 import hothX_image5 from '../../images/gallery-carousel/hoth-X/image5.jpg';
 
+import hothXI_image1 from '../../images/gallery-carousel/hoth-XI/image1.jpg';
+import hothXI_image2 from '../../images/gallery-carousel/hoth-XI/image2.jpg';
+import hothXI_image3 from '../../images/gallery-carousel/hoth-XI/image3.jpg';
+import hothXI_image4 from '../../images/gallery-carousel/hoth-XI/image4.jpg';
+import hothXI_image5 from '../../images/gallery-carousel/hoth-XI/image5.jpg';
+import hothXI_image6 from '../../images/gallery-carousel/hoth-XI/image6.jpg';
+import hothXI_image7 from '../../images/gallery-carousel/hoth-XI/image7.jpg';
+import hothXI_image8 from '../../images/gallery-carousel/hoth-XI/image8.jpg';
+import hothXI_image9 from '../../images/gallery-carousel/hoth-XI/image9.jpg';
+
```

Comment:
In the photo carousel here, I think we only need to display the pictures of the HOTH XI winners-- so it's probably OK to remove the ones that feature just the officers or anything that isn't the winners holding their certificates.